src/index.js

```diff
@@ -8,9 +8,10 @@ export {
     default as ReducerRegistry,
     applyReducerHash,
     dispatchActionsToStore,
-    reduxRegistery
+    reduxRegistery,
```

Comment:
@karelhala : I suggest replacing all "Registery" (notice the **E**) with "Registry". People will fight with this in their editors if it stays as it is ;-)